packages/application-shell/src/components/user-settings-menu/user-settings-menu.js

```diff
@@ -87,11 +86,23 @@ const renderLabel = (menu, applicationLanguage) => {
 };
 
 const UserSettingsMenuBody = props => {
-  const menuLinks =
-    (props.applicationsMenuQuery &&
-      props.applicationsMenuQuery.applicationsMenu &&
-      props.applicationsMenuQuery.applicationsMenu.appBar) ||
-    [];
+  const applicationsMenu = useApplicationsMenu({
+    queryOptions: {
+      // We can assume here that the navbar already fetched the data, since this
+      // component gets rendered only when the user opens the menu
+      fetchPolicy: 'cache-only',
+      onError: reportErrorToSentry,
```

Comment:
This is also interesting and I think we should do that more often. When we don't specifically handle the apollo `error`, we can just bind the error handler with our sentry reporter.